plugins/coverbrowser.koplugin/main.lua

```diff
@@ -126,55 +132,183 @@ function CoverBrowser:addToMainMenu(menu_items)
     end
     sub_item_table[#modes].separator = true
     table.insert(sub_item_table, {
-        text = _("Use this mode everywhere"),
+        text = _("Hide book covers"),
+        enabled_func = function()
+            return not fc.display_mode_type == false
```

Comment:
We usually write
```Lua
return fc.display_mode_type and true or false
```

Reply:
what about `return fc.display_mode_type == false` ?

Reply:
That's opposite to what you need.

Reply:
Yes i know, what i am asking is: would you write that in a different way too?

Reply:
For this case it can be
`return fc.display_mode_type ~= nil`